gnocchi/cli/metricd.py

```diff
@@ -85,6 +86,9 @@ def run(self):
             with utils.StopWatch() as timer:
                 try:
                     self._run_job()
+                except ConnectionError as ce:
+                    LOG.debug("Redis server closed connection. Retrying.")
+                    raise tenacity.TryAgain(ce)
```

Comment:
You want to ignore the exception then, the loop will retry by itself. No need to use tenacity.

Reply:
done (in the following patch)